src/core/tests/Test_CoreMain.py

```diff
@@ -1143,6 +1168,624 @@ def __check_telemetry_events(self, runtime):
+            substatus_file_data = json.load(file_handle)
+
+        self.assertTrue(len(json.dumps(substatus_file_data)) < Constants.StatusTruncationConfig.AGENT_FACING_STATUS_FILE_SIZE_LIMIT_IN_BYTES)
+
+        # Assert assessment truncation
+        self.__assert_truncated_status_output(substatus_file_data[0]["status"]["substatus"][0], Constants.PATCH_ASSESSMENT_SUMMARY, Constants.STATUS_WARNING)
+        # Assert assessment tombstone
+        self.__assert_truncated_assessment_tombstone(substatus_file_data[0]["status"]["substatus"][0], patch_count_for_assessment + 2)
+        # Assert assessment truncated error
+        self.__assert_truncated_error(substatus_file_data[0]["status"]["substatus"][0], error_count=0)
+
+        truncated_assessment_msg = json.loads(substatus_file_data[0]["status"]["substatus"][0]["formattedMessage"]["message"])
+        truncated_assessment_activity_id = truncated_assessment_msg['assessmentActivityId']
+        truncated_assessment_reboot_pending = truncated_assessment_msg['rebootPending']
+        truncated_assessment_crit_patch_count = truncated_assessment_msg['criticalAndSecurityPatchCount']
+        truncated_assessment_other_patch_count = truncated_assessment_msg['otherPatchCount']
+        truncated_assessment_start_time = truncated_assessment_msg['startTime']
+        truncated_assessment_last_modified_time = truncated_assessment_msg['lastModifiedTime']
+        truncated_assessment_started_by = truncated_assessment_msg['startedBy']
+
+        # validate all assessment other fields in the message object are equal in both status files
+        self.assertEqual(assessment_activity_id, truncated_assessment_activity_id)
+        self.assertEqual(assessment_reboot_pending, truncated_assessment_reboot_pending)
+        self.assertEqual(assessment_crit_patch_count, truncated_assessment_crit_patch_count)
+        self.assertEqual(assessment_other_patch_count, truncated_assessment_other_patch_count)
+        self.assertEqual(assessment_start_time, truncated_assessment_start_time)
+        self.assertEqual(assessment_last_modified_time, truncated_assessment_last_modified_time)
+        self.assertEqual(assessment_started_by, truncated_assessment_started_by)
+
+        # Assert installation truncation
+        installation_truncated_substatus = substatus_file_data[0]["status"]["substatus"][1]
+        self.__assert_truncated_status_output(installation_truncated_substatus, Constants.PATCH_INSTALLATION_SUMMARY, Constants.STATUS_WARNING)
+        # Assert installation tombstone
+        self.__assert_truncated_installation_tombstone(installation_truncated_substatus, patch_count_for_installation + 2)
+        # Assert installation truncated error
+        self.__assert_truncated_error(substatus_file_data[0]["status"]["substatus"][0], error_count=0)
+
+        truncated_installation_msg = json.loads(installation_truncated_substatus["formattedMessage"]["message"])
```

Comment:
Why are these variables used? I believe there was a previous comment regarding these variables and refactoring common code between different tests

Reply:
variables to remove, but keep the assertions. done